.github/workflows/ci.yml

```diff
@@ -451,3 +451,35 @@ jobs:
         with:
           name: artifacts
           path: ./out/artifacts
+
+
+  protobuf-schemas:
+    runs-on: ubuntu-22.04
+    needs: detect-noop
+    if: needs.detect-noop.outputs.noop != 'true'
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+
+      - name: Setup Buf
+        uses: bufbuild/buf-setup-action@v1
+      
+      - name: Lint Protocol Buffers
+        uses: bufbuild/buf-lint-action@v1
+        with:
+          input: apis
+
+      - name: Detect Breaking Changes in Protocol Buffers
```

Comment:
should there be a local part of the workflow here? e.g. `make reviewable` does this same check so devs can have confidence in their commit before opening a PR?

Reply:
It would be nice to have, but I feel a sense of urgency to have something in place and I don't want to take on teaching the Makefile and/or build submodule about `buf` right now. I've raised https://github.com/crossplane/crossplane/issues/4924 to track this.